babel.config.js

```diff
@@ -0,0 +1,7 @@
+// babel.config.js
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {targets: {node: '12'}}],
```

Comment:
Is this lib only targeting node or should it also support browsers?

Reply:
Good point. What would you recommend using here?

Reply:
I don't know, I think it's something we should define for all the Ceramic ecosystem so we can be consistent.
On one side we can target only "modern" browsers to create smaller builds, but that may exclude lots of mobile users, and on the other side we can target older browsers, but then we likely need to also add Babel's runtime/polyfills, and that can make the bundles quite large which is also bad for mobile browsing.
In any case we should follow [browserlist's best practices](https://github.com/browserslist/browserslist#best-practices), maybe we can just use the "default" list for now but I don't know what browsers/versions it excludes.

Reply:
Ok, lets keep it as is now until we set up a general config for all our repos  :)